src/explore-education-statistics-admin/src/pages/release/data/components/__tests__/ReleaseDataUploadsSection.test.tsx

```diff
@@ -461,6 +461,49 @@ describe('ReleaseDataUploadsSection', () => {
         }),
       ).toBeInTheDocument();
     });
+
+    test('does not allow deleting files when linked to an API data set', async () => {
+      releaseDataFileService.getDataFiles.mockResolvedValue([
+        { ...testDataFiles[0], isLinkedToApiDataSet: true },
+      ]);
+      releaseDataFileService.getDataFileImportStatus.mockResolvedValue(
+        testCompleteImportStatus,
+      );
+
+      const { user } = render(
+        <MemoryRouter>
+          <ReleaseDataUploadsSection
+            publicationId="publication-1"
+            releaseId="release-1"
+            canUpdateRelease
+          />
+        </MemoryRouter>,
+      );
+
+      await waitFor(() => {
+        expect(screen.getAllByTestId('accordionSection')).toHaveLength(1);
```

Comment:
Just a testing-related question - I was just wondering why you're using `getAllByTestId` here if there is only one? I can see down below that we're getting all `accordionSection` elements. Not sure if I'm missing something 🤔 or if the intention is to add more sections in the future